### PR TITLE
Testing Private for Ingress endpoints

### PR DESCRIPTION
**Pull request by pcrentsil**
Testing Implementing Private Logic to support ingress endpoint publishing strategy scope.

For testing purposes see commented code references...

Implenting Private function params in ReconcileDefaultIngressController 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: control-plane-operator/controllers/hostedcontrolplane/kas/config.go [hunk 1]
```diff
@@ -191,7 +191,8 @@ func generateConfig(p KubeAPIServerConfigParams, version semver.Version) *kcpv1.
 	args.Set("service-account-lookup", "true")
 	args.Set("service-account-signing-key-file", cpath(kasVolumeServiceAccountKey().Name, pki.ServiceSignerPrivateKey))
 	args.Set("service-node-port-range", p.NodePortRange)
-	args.Set("shutdown-delay-duration", "10s")
+	args.Set("shutdown-delay-duration", "70s")
+	args.Set("shutdown-send-retry-after", "true")
```
Comment (hasueki) [inline control-plane-operator/controllers/hostedcontrolplane/kas/config.go:195]: This looks like an outdated change. You most likely need to rebase your branch.

File: control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/params.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ type IngressParams struct {
 	IngressSubdomain string
 	Replicas         int32
 	PlatformType     hyperv1.PlatformType
+	IsPrivate        bool
 }
 
 func NewIngressParams(hcp *hyperv1.HostedControlPlane, globalConfig globalconfig.GlobalConfig) *IngressParams {
```
Comment (hasueki) [inline control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/params.go:15]: We will need logic to set `IsPrivate` in this `NewIngressParams` function.

**Comment by pcrentsil** [comment]
Now Pointing to https://github.com/openshift/hypershift/pull/1597

**Pull request closed without merging**